book/website/reproducible-research/open/open-access.md

```diff
@@ -36,6 +36,13 @@ You can find out more about preprints at [ASAPbio](https://asapbio.org/preprint-
 (rr-open-access-journal-publishing)=
 ### Open Access Journal publishing
 
+Watch this talk by Vicky Hellon on "Brief History of scientific publishing, and where do we go from here?", where she provides an overview of how different models of scientific publication look like, and how we can take an informed approach to Open Access.
+
+<div class="video-container">
+  <iframe src="https://www.youtube.com/embed/Mycp7SzBWB0" width="640" height="360" frameborder="0" allow="autoplay; fullscreen; picture-in-picture" allowfullscreen></iframe>
+  <p><a href="https://www.youtube.com/embed/Mycp7SzBWB0">History of  Scientific Publishing - Where do we go from here??</a> from <a href="https://vimeo.com/orcidvideos">ORCID</a> on <a href="https://vimeo.com">Vimeo</a>.</p>
+</div>
+
```

Comment:
Slides for this talk can be found [here](https://doi.org/10.5281/zenodo.6516871).